vertical-pod-autoscaler/e2e/v1/actuation.go

```diff
@@ -445,6 +493,33 @@ func testEvictsPods(f *framework.Framework, controller *autoscaling.CrossVersion
 	gomega.Expect(err).NotTo(gomega.HaveOccurred())
 }
 
+func testEvictsSingletonPod(f *framework.Framework, controller *autoscaling.CrossVersionObjectReference) {
```

Comment:
Please rename or break into two helpers (setup & verify no eviction + update & verify eviction).

I wanted to suggest adding a test checking that we don't evict singleton pod unless configured to do it.

Then I read the body of this function and learned that we do test that.

`testEvictsSingletonPodOnlyAfterConfiguredTo` ?

Reply:
I split into two tests & moved to parametrised test suite (I didn't want to copy & paste 3 tests for each controller). PTAL.